data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -21,7 +23,8 @@
 
 TABLE_INFORMATION_SCHEMA = JINJA_ENV.from_string(  # language=JINJA2
     """
-SELECT *
+SELECT * except(is_generated, generation_expression, is_stored, is_hidden,
+is_updatable, is_system_defined, clustering_ordinal_position)
 FROM `{{project}}.{{dataset}}.INFORMATION_SCHEMA.COLUMNS`
```

Comment:
Thank you. I believe this makes the result identical for both client and no-client situations?

Reply:
I think this is fine. As long as the extra columns aren't used anywhere, this works as expected.